src/SIL.XForge.Scripture/Services/AnonymousService.cs

```diff
@@ -29,7 +29,7 @@ public async Task<AnonymousShareKeyResponse> CheckShareKey(string shareKey)
         // Ensure the key hasn't been used by another recipient
         if (validShareKey.ShareKey.RecipientUserId != null)
         {
-            throw new ForbiddenException();
+            throw new DataNotFoundException("key_already_used");
```

Comment:
I'm using `DataNotFoundException` because it's being used at https://github.com/sillsdev/web-xforge/blob/3498a653361fa67cddffb7a6035c17d44bace2fc/src/SIL.XForge.Scripture/Services/SFProjectService.cs#L771 for the same error message.